admin/pageaudit/report/models.py

```diff
@@ -66,18 +67,36 @@ class LighthouseRun(models.Model):
     thumbnail_image = models.TextField(blank=True, null=True)
     time_to_first_byte = models.PositiveIntegerField(default=0)
     total_byte_weight = models.PositiveIntegerField(default=0)
-    
+
     ## REMOVE THIS after new user-timing models are POPULATED WITH THE DATA.
     masthead_onscreen = models.PositiveIntegerField(default=0)
 
-    
+
     class Meta:
         ordering = ['-created_date']
-        
+
     def __str__(self):
         return 'perf: %s - requests: %s' % (self.performance_score, self.number_network_requests,)
 
 
+class UrlOwner(models.Model):
```

Comment:
Also added this UrlOwner model we can manually populate as needed for now - long term we will try to auto-populate this from web page metadata